dom.js

```diff
@@ -69,4 +75,61 @@ export function playBackgroundMusic(isMuted) {
     bgMusic.loop = true
     bgMusic.play()
     bgMusic.classList.add('musicPlaying')
+}
+
+export function movingBasketSound(isMuted) {
+    const basketSound = document.getElementById(EL_IDS.basketSound)
+    if (!basketSound) return
+    basketSound.muted = isMuted
+    basketSound.currentTime = 0
+    basketSound.play()
+}
+
+export function rocketTakeOffSound(isMuted) {
+    const rocketTakeOffSound = document.getElementById(EL_IDS.rocketTakeOffSound)
+    if (!rocketTakeOffSound) return
+    rocketTakeOffSound.muted = isMuted
+    rocketTakeOffSound.currentTime = 0
+    rocketTakeOffSound.play()
+}
+
+export function rocketExplodeSound(isMuted) {
+    const lifeLostSound = document.getElementById(EL_IDS.lifeLostSound)
+    if (!lifeLostSound) return
+    lifeLostSound.muted = isMuted
+    lifeLostSound.currentTime = 0
+    lifeLostSound.play()
+}
+
+
+export function levelUpSound(isMuted) {
+    const levelUpSound = document.getElementById(EL_IDS.levelUpSound)
+    if (!levelUpSound) return
+    levelUpSound.muted = isMuted
+    levelUpSound.currentTime = 0
+    levelUpSound.play()
+}
+
+export function SpawnFallingObjectSound(isMuted) {
+    const spawnFallingObjectSound = document.getElementById(EL_IDS.spawnFallingObjectSound)
+    if (!spawnFallingObjectSound) return
+    spawnFallingObjectSound.muted = isMuted
+    spawnFallingObjectSound.currentTime = 0
+    spawnFallingObjectSound.play()
+}
+
+export function catchFallingObjectSound(isMuted) {
+    const catchFallingObjectSound = document.getElementById(EL_IDS.catchFallingObjectSound)
+    if (!catchFallingObjectSound) return
+    catchFallingObjectSound.muted = isMuted
+    catchFallingObjectSound.currentTime = 0
+    catchFallingObjectSound.play()
+}
+
+export function gameOverSound(isMuted) {
+    const gameOverMusic = document.getElementById(EL_IDS.gameOverMusic)
+    if (!gameOverMusic) return
```

Comment:
All these functions are structured the exact same way. So you could "DRY-out" a lot of this code by creating a single function that accepts both `isMuted` and `elementId` then each of these functions can just call that, passing in the specific `EL_IDS` they need to operate on.

Again, not super important

Reply:
Thanks for pointing that out. I went and did what you said. and added most of the functions to the state mutators with one function